src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1586,6 +1593,14 @@ private void ApplyUpdate()
                 ResetForDeltaExecution();
                 runnerCore.Options.ApplyUpdate = true;
                 Execute(true);
+
+                // Execute() will push a stack frame in SetupAndBounceStackFrame().
+                // In normal execution, that stack frame will pop in RETB. But in
+                // ApplyUpdate(), there is no RETB instruciton, so need to manually
+                // cleanup stack frame.
+                StackValue restoreFramePointer = runtimeCore.RuntimeMemory.GetAtRelative(ProtoCore.DSASM.StackFrame.kFrameIndexFramePointer);
```

Comment:
I hope the next exercise is to turn `kFrameIndexFramePointer` into `FrameIndexFramePointer` ;)